src/open_inwoner/pdc/admin/product.py

```diff
@@ -93,6 +93,14 @@ def display_categories(self, obj):
 
     display_categories.short_description = "categories"
 
+    def get_form(self, request, obj=None, **kwargs):
+        form = super().get_form(request, obj, **kwargs)
+
+        if obj:
+            form.base_fields["link"].widget.attrs.update({"data-product-id": obj.id})
+            form.base_fields["form"].widget.attrs.update({"data-product-id": obj.id})
+        return form
+
```

Comment:
Is this still necessary now the button doesn't need to carry the ID?

Reply:
Correct, I will fix it.